nav-app/src/app/tabs/tabs.component.scss

```diff
@@ -188,6 +188,43 @@ input[type=file]{
         height: 2ex;
         padding: 5px;
     }
+
+    .banner {
+        background-color: color(primary);
+        margin: 7px 0;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 5px 15px;
+        border-radius: 5px;
+        text-align: left;
+        color: on-color(primary);
+        font-size: 14px;
+        font-family: Roboto, "Helvetica Neue", sans-serif;
+        .bannerContent {
+            padding-right: 15px;
+        }
+        .bannerLinks {
+            display: flex;
+            align-items: center;
+            border-left: 2px solid #ffffffad;
+            padding-left: 15px;
+        }
+        .helpIcon {
+            height: 18px;
+            padding-left: 15px;
+        }
+        ::ng-deep a, a, ::ng-deep * {
+            display: inline;
+            color: on-color(primary);
+            text-decoration-line: none;
+            text-decoration-color: on-color(primary);
+        }
```

Comment:
This removes any text styling for hyperlinks, which breaks the banner if we wanted to embed multiple links. 
For instance:

`&#9888; ATT&CK version 9 has been released! See <a href=\"google.com\">the changelog</a> or <a href=\"google.com\">visit the website</a>.`

> &#9888; ATT&CK version 9 has been released! See <a href="google.com">the changelog</a> or <a href="google.com">visit the website</a>.



Which displays as:

<img width="630" alt="image" src="https://user-images.githubusercontent.com/28868888/116285235-f2a9aa00-a75b-11eb-8f53-c90f5865633c.png">

The only way to tell where the hyperlinks are is by hovering with the mouse. Ideally `a` elements should be styled somehow -- maybe keeping the underline? It's a bit funky because of the orange background.

Reply:
I believe `::ng-deep` is a deprecated feature. The correct way to do this in recent versions of Angular is through [encapsulation](https://github.com/mitre-attack/attack-navigator/blob/master/nav-app/src/app/datatable/data-table.component.ts#L26).